stores/gaugeStore.js

```diff
@@ -604,12 +606,16 @@ class Store {
 
     for (let i = 0; i < project.gauges.length; i++) {
       project.gauges[i].balance = (gaugesData[i].balanceOf / 10 ** project.gauges[i].lpToken.underlyingDecimals) * project.gauges[i].lpToken.conversionRate;
-      project.gauges[i].workingBalance = gaugesData[i].workingBalanceOf;
+      project.gauges[i].workingBalance = gaugesData[i].workingBalanceOf; //effective balance after boost applied
+      const boostAppliedAmount = project.gauges[i].workingBalance;
+      const workingBalanceNormalized = (boostAppliedAmount / 10 ** project.gauges[i].lpToken.underlyingDecimals) * project.gauges[i].lpToken.conversionRate;
+
       project.gauges[i].workingSupply = gaugesData[i].workingSupply;
       project.gauges[i].rawBalance = gaugesData[i].balanceOf;
 
       project.gauges[i].remainingBalance = userRemainingStake(
-        project.gauges[i].balance,
+        // project.gauges[i].balance,
+        workingBalanceNormalized,
```

Comment:
I think this is not correct. we still want to use the real balance value not the working balance here

What is needed is to modify the implementation of `userRemainingStake` to take into account the user boost instead of using the applied boost

currently this function works under the assumption that use is at max boost. In that case balance and workingBalance is equal

but this not a good assumption and is the cause of the issue highlighted here.

the function returns the following value `maxStakeBalance - balance`
instead it should do `maxStakeBalance / (boost * 0.4) - balance`

`boost` is already calculated in the `gauge` object